08-rsf-prep.qmd

```diff
@@ -1,3 +1,193 @@
 ---
 title: "Generate random sample points for RSF"
 ---
+
+
+## Overview:
+
+In this module we use our prepared spatial data to generate the input datafiles to conduct a resource selection function analysis. This includes: 
+
+-   generate a set of "available" or "background points" from our study area.
+-   extract spatial information for presence and available point locations
+-   export as table for future use
+-   use the extracted data to provide further graphics and summary options
+
+
+## Background: Resource Selection Functions
+
+Now that we have a cleaned and standardized dataset for the Scott Herd Caribou's we can prepare the data for further analysis.
+
+Resource Selection Functions are a common method used to assess what is driving patterns of animal habitat preference. This process uses information or covariates (i.e. landscape attribute features) for locations where animals are present and compares them to the same features where animals are absent. In this way we can gather some information on what conditions (i.e. landscape, aspect, distance from road, etc) which contribute to higher habitat use. You can find more references for Resource Selection Functions below.
+
+While in this example we are concentrating on future analysis, the process of extracting background information for known locations can also provide meaningful summary statistics, i.e. proportion of locations within a specific BEC zone.
+
+
+## 1. Generate availalble location points.
+
+We will generate a simple set of "available" based on the geographic distribution of the study area. Note you can limit this area in which you generate "background" points, using more sophisticated methods, such as within a kernal density or home range estimate.
+
+Lets start by reading in the libraries we will use.
+
+```{r, eval = FALSE}
+
+library(dplyr)
+library(terra)
+library(sf)
+library(mapview)
+library(ggplot2)
+
+```
+
+Next, read in the standardized data points for the Scott herd. If needed we can transform to BC Albers projection (EPGS:3005) to match our raster stack prepared in the previous step. 
+
+```{r, eval = FALSE}
+
+# read in the aoi template 
+
+template <- rast(file.path("clean_data","template.tif"))
+
+
+# read in points 
+
+bou_pts <- st_read(file.path("clean_data/scott_herd_subset.gpkg")) 
```

Comment:
For me this is only a csv currently

Reply:
ok roger, i think i need to rerun from the start to clear out the old stuff.